src/AzureAppConfiguration.ts

```diff
@@ -3,4 +3,4 @@
 
 export type AzureAppConfiguration = {
     // methods for advanced features, e.g. refresh()
-} & ReadonlyMap<string, unknown>;
+} & ReadonlyMap<string, any>;

```

Comment:
Note:
As discussed in last review meeting, here change `unknown` to `any` for easier use of the library. Otherwise, you always need to explicitly do typecasting downstream.